### DISCUSSION TITLE
GET http://localhost:3000/uuidv4 404 (Not Found)

### DISCUSSION THREAD

**Discussion by throne1986** [GrapesJS/grapesjs#3856] [Q&A]
Everything was working fine until now when I open the editor it keeps popping up errors as the image below shows .

Any suggestion what do I need to do to solve this issue? @artf 

![image](https://user-images.githubusercontent.com/27904724/136366631-4e3edfd9-4945-4db8-9066-bcd04d3747c9.png)

**Comment by artf** [comment]
I guess this is something related to your application, might be a missing image.

**Reply by throne1986** [reply]
> I guess this is something related to your application, might be a missing image.

Yes ur right it was an image problem, but now I have this error, any suggestions on what might be causing this issue?
![image](https://user-images.githubusercontent.com/27904724/136371466-f6f5e15f-4e4d-412a-ba16-45a881ed01cd.png)

**Reply by artf** [reply]
Might be some chrome extension https://stackoverflow.com/questions/54126343/how-to-fix-unchecked-runtime-lasterror-the-message-port-closed-before-a-respon
